### PR TITLE
delete repository tag

### PR DESCRIPTION
**Pull request by gokhanm**
The gitlab api supports deleting tag in repository

http://doc.gitlab.com/ce/api/tags.html

### CODE REVIEW

File: gitlab/__init__.py [hunk 1]
```diff
@@ -1415,6 +1415,25 @@ def createrepositorytag(self, project_id, tag_name, ref, message=None):
         else:
             return False
 
+    def deleterepositorytag(self, project_id, tag_name):
+
```
Comment (FredrikWendt) [inline gitlab/__init__.py]: Removing this empty line would be cleaner imho.

**Comment by gokhanm** [comment]
You are right. I have removed the line.

**Comment by FredrikWendt** [comment]
Functionality wise, this LGTM. Code style wise, there's another double empty lines I would remove, but if you look through the file, there's a mix of functions with empty lines to separate "logical parts" of the functions, and functions where it's all crammed up with no lines separating logical parts (typically: prep request data, make and send request, process response).

File: gitlab/__init__.py (hunk 1 above)
Comment (Sudo-Kid) [reply gitlab/__init__.py]: I'm with Fredrik on this one. The docstring should be directly under the functions declaration.

**Comment by Sudo-Kid** [comment]
I'll look this over more this weekend and see about getting it merged in.

**Pull request merged**